index.html

```diff
@@ -32,6 +33,9 @@
 				<th style="width: 45%;">Go</th>
 			</tr>
 		</table>
+		<button class="clipboard" data-clipboard-target="#output">
+			<img src="resources/images/clippy.svg" alt="Copy to clipboard">
+		</button>
```

Comment:
Can we move the button to be inline with the word "Go" above the output box? Instead of on top of the scrollbar...